### PR TITLE
WAZO-2349 - added timestamp in order to detect any delay from any event 

### PR DESCRIPTION
**Pull request by oxynor**

### CODE REVIEW

**Comment by oxynor** [comment]
recheck

File: xivo_bus/tests/test_marshaler.py [hunk 1]
```diff
@@ -82,6 +81,14 @@ def test_given_valid_message_when_unmarshal_message_then_return_object(self):
         assert_that(result.value, equal_to(1234))
         assert_that(result.metadata, equal_to({'origin_uuid': 'some-origin'}))
 
+    @patch("xivo_bus.marshaler.datetime")
```
Comment (bloom1) [inline xivo_bus/tests/test_marshaler.py:84]: Consistency: we try to use the same string quote style across the code. In this case, it should be single quotes (`'str'`)

**Review by bloom1 — CHANGES_REQUESTED** [review]
Good work :smiley:

**Comment by bloom1** [comment]
@oxynor up

**Review by bloom1 — APPROVED** [review]

**Pull request merged**